frosthaven_assistant_server/Dockerfile

```
@@ -0,0 +1,23 @@
+# Specify the Dart SDK base image version using dart:<version> (ex: dart:2.12)
+FROM dart:stable AS build
+
+# Resolve app dependencies.
+WORKDIR /app
+COPY pubspec.* ./
+RUN dart pub get
+
+# Copy app source code and AOT compile it.
+COPY . .
+# Ensure packages are still up-to-date if anything has changed
+RUN dart pub get --offline
+RUN dart compile exe lib/main.dart -o bin/server
```

Dart doesn't appear to make intermediate directories when specifying an output path.

```suggestion
RUN mkdir -p bin &&\
    dart compile exe lib/main.dart --output=bin/server
```

This would resolve a build failure that occurs with the image:
```
 > [build 7/7] RUN dart compile exe lib/main.dart -o bin/server:
1.861 Error: AOT compilation failed
1.862 PathNotFoundException: Cannot open file, path = '/app/bin/server' (OS Error: No such file or directory, errno = 2)
------
```